packages/compass-connection-import-export/src/components/export-modal.tsx

```diff
@@ -6,10 +6,10 @@ import {
   FormFieldContainer,
   FormModal,
   openToast,
+  SelectTable,
```

Comment:
Moved out of connection-import-export as we now have two places where this UI is used